lib/Core/Executor.cpp

```diff
@@ -2950,6 +2950,11 @@ void Executor::callExternalFunction(ExecutionState &state,
       assert(success && "FIXME: Unhandled solver failure");
       (void) success;
       ce->toMemory(&args[wordIndex]);
+      ObjectPair op;
+      if(ce->getWidth() == Context::get().getPointerWidth()
```

Comment:
Can you add a comment here saying that you check to see if the argument is a pointer?